tests/test_io.py

```diff
@@ -1365,3 +1347,30 @@ def test_all_read_dtype_backend() -> None:
         ),
         TextFileReader,
     )
+
+    if TYPE_CHECKING:
```

Comment:
Not sure why this is in an `if TYPE_CHECKING:` block.  I think this should work at runtime.

Reply:
`sqlite3` does not support `read_sql_table` so did it like this

Reply:
can you copy the comment that was in the other test for `read_sql_table()` into here?

Reply:
Done